src/engine/engine.cpp

```diff
@@ -434,23 +434,23 @@ const char* DivEngine::getSystemName(DivSystem sys) {
     case DIV_SYSTEM_VRC6:
       return "Konami VRC6";
     case DIV_SYSTEM_OPLL:
-      return "Yamaha OPLL";
+      return "Sega FM Sound Unit";
```

Comment:
Sounds like the most popular option, but I think the OPLL was used in other places as well...

Reply:
sadly. NTSC-J Master System, MSX2+ and Turbo-R, MSX MUSIC-standard expansion cartridges form older MSXs, several PortaTone home keyboards...

Reply:
Yamaha OPLL....